src/main/java/com/textmessenger/service/DialogServiceImpl.java

```diff
@@ -18,12 +20,15 @@
   private final DialogRepository dialogRepository;
   private final UserRepository userRepository;
   private final NotificationService notificationService;
+  private SimpMessagingTemplate simpMessagingTemplate;
+  private final String wsPath = "/queue/messages";//NOSONAR
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make this final field static too. ([squid:S1170](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1170))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=175&issues=AWXEvuC-_eEZPtUmi2K8&open=AWXEvuC-_eEZPtUmi2K8)

[ISSUE_KEY]: AWXEvuC-_eEZPtUmi2K8